dkpro-core-opennlp-asl/pom.xml

```diff
@@ -486,7 +486,7 @@
       <dependencies>
       <dependency>
 		  <groupId>de.tudarmstadt.ukp.dkpro.core</groupId>
-		  <artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-model-tagger-af-afribooms</artifactId>
+		  <artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-model-tagger-af-afribooms-perceptron</artifactId>
```

Comment:
The test dependency should be **either** in the standard dependency section **or** in the `use-full-resources` profile - not in both. I would suggest to retain it only in the `use-full-resources` profile to reduce PR build times.